src/lib/components/common/PlausibleTracker.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import Plausible from "plausible-tracker";
+
+  import { getContext, onMount } from "svelte";
+  import type { Writable } from "svelte/store";
+  import { afterNavigate } from "$app/navigation";
+  import { browser } from "$app/environment";
+  import type { User } from "@/lib/api/types";
+  import type { Maybe } from "@/api/types/common";
+
+  let plausible;
+  const user: Writable<Maybe<User>> = getContext("me");
+
+  onMount(() => {
+    if (browser) {
+      plausible = Plausible();
+    }
+  });
+
+  afterNavigate(() => {
+    if (user && browser) {
```

Comment:
We also want to check if we're in an embed request. There's a value set in the root layout and passed down through context.

Calling `getContext('embed')` will be true if we're in an embed.

We don't want to inject any tracking code into embeds.

Reply:
I think this is solved by not relying on `AppLayout` in root—see some of the route filesystem changes in #657.